src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/Persons/PersonsControllerNegativeTests.cs

```diff
@@ -31,13 +31,43 @@ public async Task CreateSavedFilter_AsHacker_ShouldReturnBadRequest_WhenUnknownP
 
         [TestMethod]
         public async Task CreateSavedFilter_AsHacker_ShouldReturnForbidden_WhenPermissionMissing()
-            => await PersonsControllerTestsHelper.CreateSavedFilter(
+            => await PersonsControllerTestsHelper.CreateSavedFilterAsync(
                 UserType.Hacker,
                 TestFactory.PlantWithAccess,
                 "title",
                 "criteria",
                 true,
                 HttpStatusCode.Forbidden);
         #endregion
+
+        #region DeleteSavedFilter
+        [TestMethod]
+        public async Task DeleteSavedFilter_AsAnonymous_ShouldReturnUnauthorized()
+            => await PersonsControllerTestsHelper.DeleteSavedFilterAsync(
+                UserType.Anonymous,
+                TestFactory.UnknownPlant,
+                1,
+                "rowVersion",
```

Comment:
Use a valid rowVersion (so an invalid rowVersion do not mess up the issue under test). Same for tests below